DDI.csv

```diff
@@ -1,451 +1,451 @@
-16978,Edwards 1987,https://pubmed.ncbi.nlm.nih.gov/3802708/,VER-QUI-DDI_120mg,Quinidine,Verapamil,PO,PO,,Plasma,1.25,,,,0.96,,,,0,12,72.5,84.5,h,1,AUCR and CmaxR derived from the digitized plasma profiles,120,111.10,mg,PO,S0-T8-R12,h,TID,,verapamil,,,,,,,,
-16949,Abolfathi 1993,https://pubmed.ncbi.nlm.nih.gov/8371133/,QUI-MEX-Control_PM,Mexiletine,Quinidine,PO,PO,,Plasma,0.65,,,,0.63,,,,0,48,48,96,h,1,!! DDGI Ratios PM vs. NM !! DDGI AUCR and DDGI CmaxR derived from the digitized plasma profiles,50,41.44,mg,PO,S0-T6-R16,h,QID,,quinidine sulfate,,,,,,,,
+"ID","Study ID","Reference","Grouping","Victim","Perpetrator","Route Victim","Route Perpetrator","Mechanism","Compartment","AUCR Avg","AUCR AvgType","AUCR Var","AUCR VarType","CmaxR Avg","CmaxR AvgType","CmaxR Var","CmaxR VarType","t_placebo_0","t_placebo_end","t_treatment_0","t_treatment_end","t Unit","R Dose correction factor","Comment","Dose","Dose free API","Dose Unit","Route","Times of Administration","Times Unit","Description","Comment Regimen","Administered form","Formulation type","Infusion duration [min]","Water volume ingested for drug intake [mL]","Comment on administration/formulation","Fasted/Fed state","Duration of fasting before drug administration [h]","Duration of fasting after drug administration [h]","Comment on food intake"
+50,"Ahonen 1995","http://www.ncbi.nlm.nih.gov/pubmed/8527290","with Perpetrator (Itraconazole)","Midazolam","Itraconazole","PO","PO","Reversible_Inhibition","Plasma",5.74509803921569,"Approx. Avg",,,2.55882352941176,"Approx. Avg",,,"0","inf","74","inf","h",,,"100",,"mg","PO","(S-0,T-24,R-4)","h","MD OD (4 days)","OD",,"Sporanox",,,"Sporanox Janssen",,,,
+54,"Backman 1996","https://www.ncbi.nlm.nih.gov/pubmed/8549036","with Perpetrator (Rifampicin)","Midazolam","Rifampicin","PO","PO","Induction","Plasma",0.041,"Arith.mean",,,0.0618181818181818,"Approx. CmaxR",,,"0","inf","113","inf","h",1,,"600",,"mg","PO","0-24-48-72-96","h","MD OD (5 days)",,,"300 mg capsules, Rimapen Orion",,,,,,,
+56,"Backman 1998","https://www.ncbi.nlm.nih.gov/pubmed/9591931","Phase IV (during Perpetrator (Rifampicin))","Midazolam","Rifampicin","PO","PO","Induction","Plasma",0.023,"arith. mean","6.0000000000000001E-3","Arith. SEM",0.054,"Arith. Mean","1.2E-2","Arith. SEM","0","inf","113","185","h",1,,"600",,"mg","PO","0-24-48-72-96","h","MD OD (5 days)",,,"300 mg capsules, Rimapen Orion",,,,"Fasted",,,
+57,"Backman 1998","https://www.ncbi.nlm.nih.gov/pubmed/9591931","Phase V (4 days after Perpetrator (Rifampicin))","Midazolam","Rifampicin","PO","PO","Induction","Plasma",0.132,"arith. mean","2.8000000000000001E-2","Arith. SEM",0.202,"Arith. Mean","4.2000000000000003E-2","Arith. SEM","0","inf","185","inf","h",1,,"600",,"mg","PO","0-24-48-72-96","h","MD OD (5 days)",,,"300 mg capsules, Rimapen Orion",,,,"Fasted",,,

```

Comment:
Still not happy with the new numerical representation.
e.g. before:
`... arith. mean,0.028,Arith. SEM,0.202,Arith. Mean,0.042 ...`
now:
`... "arith. mean","2.8000000000000001E-2","Arith. SEM",0.202,"Arith. Mean","4.2000000000000003E-2" ...`

should we fix it?